FirebaseFunctions/Sources/Functions.swift

```diff
@@ -397,29 +404,12 @@ enum FunctionsConstants {
       if let error {
         completion(.failure(error))
       } else {
-        let url = self.urlWithName(name)
-        self.callFunction(url: URL(string: url)!,
-                          withObject: data,
-                          options: options,
-                          timeout: timeout,
-                          context: context,
-                          completion: completion)
-      }
-    }
-  }
+        let url = switch endpoint {
+        // Should we instead complete with an error if the URL is `nil`?
```

Comment:
It may be helpful to let it crash since the function name's correctness should be enforced during development. In practice, I think the only way to get here would be to give a string function name that cannot be converted to a URL. I'm thinking that whatever this could be would probably not be a valid function name to be used in the console.